tests/cases/conformance/salsa/typeFromPrototypeAssignment2.ts

```diff
@@ -0,0 +1,46 @@
+// @noEmit: true
+// @allowJs: true
+// @checkJs: true
+// @Filename: a.js
+// @strict: true
+
+// non top-level:
```

Comment:
Why not just change the old test? This seems identical to that but not top-level.

Reply:
I prefer duplicating tests because even though this bug didn't depend on the isToplevel code path, one of the two tests may catch some future regression there.